playground/mocks/config/responseConfig.js

```diff
@@ -246,8 +247,11 @@ const authn = {
 //enroll QR Okta Verify
 const enrollQROktaVerify  = {
   '/api/v1/authn': [
-    'mfa-enroll-qr-okta-verify'
```

Comment:
is this json response not good for `enrollQROktaVerify` flow? 
I think you should add a different `const` with the responses you need for `enrollOVManually`